### PR TITLE
Create TUTORIAL.md

### PR DESCRIPTION
**Pull request by avivaschmitz**
I have contributed a tutorial that walks a user through the steps necessary to set up Spotipy and start using it for the very first time to complete a simple task. The intended audience is new users and potential contributors who have little experience coding and no experience with the Spotify API. 

This tutorial is a necessary addition to Spotipy because there is very little existing documentation for brand new users. The [README.md](https://github.com/plamere/spotipy/blob/master/README.md) file contains installation instructions and a “Quick Start” section, but they are severely lacking in detail and are far from complete. For example, they don’t contain any information on how to connect Spotipy to your Spotify developer account, how to set up a Spotify app that is compatible with Spotipy, or how to use Spotipy to accomplish a basic task. The new tutorial that I have created covers all of this.

### CODE REVIEW

**Review by Shide — APPROVED** [review]
LGTM

File: TUTORIAL.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+D. Close main.py and return to the directory that contains main.py. You can then run your app by entering the following command: python main.py
+
+E. You may see a window open in your browser asking you to authorize the application. Do so - you will only have to do this once.
+
+F. Return to your terminal - you should see all of Taylor's albums printed out there.
+
+
+
+
+
```
Comment (stephanebruckert) [inline TUTORIAL.md:86]: please remove these empty lines

**Review by stephanebruckert — CHANGES_REQUESTED** [review]
Can you add a link to TUTORIAL.md in the README?

https://github.com/plamere/spotipy/blob/master/README.md#reporting-issues

You could write:

> For common questions please check our [FAQ](FAQ.md) or [beginner tutorial](TUTORIAL.md).

**Comment by stephanebruckert** [comment]
I ended up merging https://github.com/plamere/spotipy/pull/872

**Pull request closed without merging**